data/HoldUpFjord.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "Dominick J Monaco",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Conservation Tech turned Software Engineer. Interested in contributing to community, climate change solutions, and dinner parties",
+  "avatar": "https://github.com/HoldUpFjord.png",
+   "tags": [
+    "100Devs",
+    "Green-tech",
+    "Javascript",
+    "Node.js",
+    "Tailwind",
+    "React",
+    "MongoDb"
+    ],
+  
+  "links": [
+    {
+      "name": "Follow my stream of consciousness",
+      "url": "https://twitter.com/JackMonaco1",
+      "icon": "FaTwitter"
+    }, {
+      "name": "Connect with me on Linkdln",
+      "url": "https://www.linkedin.com/in/dominick-j-monaco/",
+      "icon": "FaLinkdin"
```

The icon name seems to be wrong.
```suggestion
      "icon": "FaLinkedin"
```

---

Ah whoops. It is fixed now.